waltz-web/src/main/java/com/khartec/waltz/web/json/SurveyQuestionInfo.java

```diff
@@ -0,0 +1,19 @@
+package com.khartec.waltz.web.json;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.khartec.waltz.model.survey.SurveyQuestion;
+import com.khartec.waltz.model.survey.SurveyQuestionDropdownEntry;
+import org.immutables.value.Value;
+
+import java.util.List;
+import java.util.Optional;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableSurveyQuestionInfo.class)
+@JsonDeserialize(as = ImmutableSurveyQuestionInfo.class)
+public abstract class SurveyQuestionInfo {
+
+    public abstract SurveyQuestion question();
+    public abstract Optional<List<SurveyQuestionDropdownEntry>> dropdownEntries();
```

Comment:
Why optional as well as a list? - immutables will create an empty list if not specified in the builder